src/value_generators/include/value_generators/Pipeline.hpp

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022-present MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef HEADER_XYZ
```

Comment:
Two questions: 

1) How do I generate these crazy include guard numbers that the other files have?
2) Is this a good place for this file?

Reply:
1. `./run-genny generate-uuid-tag` although we could be much more helpful with this. The idea was that most contributors wouldn't create new header files except for their new Actors which we scaffold for them.
2. Actually if this works it's by accident. The `value_generators` cmake module [depends on bsoncxx not mongocxx](https://github.com/mongodb/genny/blob/dcd3a0527cce7afbe1aaec90b3cb488672253b5e/src/value_generators/CMakeLists.txt#L18-L25). The idea being that value_generators are "mongo agnostic" to keep their focus tight. I think the best place is actually in [conventions.hpp](https://github.com/mongodb/genny/blob/dcd3a0527cce7afbe1aaec90b3cb488672253b5e/src/gennylib/include/gennylib/conventions.hpp). (Which arguably should be named `conversions.hpp` but shrug.) It's hard to keep these kinds of utilities/conversions organized, so pushback or alternative ideas welcome.